difftest/dpi_t1rocketemu/src/interconnect.rs

```diff
@@ -192,14 +591,16 @@ impl AddressSpace {
   }
 }
 
+pub const SRAM_BASE: u32 = 0x2000_0000;
+pub const SRAM_SIZE: u32 = 0xa000_0000;
```

Comment:
How does the DDR region configured? We can statically config a region for DDR, and print this region with `--help` from the emulator.

Reply:
Actually these two constants should be named `RAM_{BASE,SIZE}` because they also represent the DRAM address mapping. To reuse the ELF across different memory latency models, the memories have to have the same mapping.

Reply:
Done in 5bc77dd5